.github/workflows/edfi.database.admin package.yml

```diff
@@ -0,0 +1,107 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: EdFi.Database.Admin Package build
```

Comment:
Please move the workflow to the ODS repo.

Reply:
Why do you want to ODS repo .  Actually code which is executing is in ODS implementation Repo

Reply:
This workflow needs to exist in the ODS repository because the filter that looks for only SQL files that have changed would only work in the ODS repo where those SQL files exist.

Reply:
Packages- EdFi.Database.Admin workflow moved to ODS repo now